cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -118,19 +118,34 @@
         <div class="{{- _classes( nav_depth, '-grid', 'three-col' if nav_item.featured_content or nav_item.nav_groups | count < 4 else '' ) -}}">
 
             {% if nav_depth > 1 %}
-            <span class="{{- _classes( nav_depth, '-overview' ) -}}">
-              {# TODO: Remove the check for '#' when real overview pages
+            <h3 class="{{ _classes( nav_depth, '-overview' ) }}
+                       {{ _classes( nav_depth, '-overview-heading' ) }}
+            ">
+                {# TODO: Remove the check for '#' when real overview pages
                        are added for the Consumer Resources and Education
                        pages. #}
                 {% if nav_overview_url %}
-                <a class="{{ 'u-link__disabled' if nav_overview_url == '#' else '' }}
-                          {{ _classes( nav_depth, '-overview-link' ) }}
-                          {{ _classes( nav_depth, '-overview-link__current' ) if nav_overview_url == request.path else '' }}"
-                    {{ '' if nav_overview_url == request.path or nav_overview_url == '#' else 'href=' + nav_overview_url | e }}>
-                    {{ nav_overview_text if nav_overview_url == '#' else nav_overview_text ~ ' Overview' }}
-                </a>
+                    {% if nav_overview_url == '#' %}
+
+                    {# only use a link element when there is an overview page to link to, otherwise we use regular heading text for context #}
+
+                    <span class="{{ _classes( nav_depth, '-overview-heading-text' ) }}">{{ nav_overview_text }}</span>
+
+                    {% else %}
+
+                    <span class="{{ _classes( nav_depth, '-overview-heading' ) }}
```

Comment:
Do we need this span? It looks like it's duplicating the classes of both the `h3` and the `a` so it probably doesn't need to stay.

<img width="617" alt="screen shot 2018-06-13 at 2 55 56 pm" src="https://user-images.githubusercontent.com/1280430/41374851-f0b40c86-6f19-11e8-9b22-d7fe25d82246.png">